vespa-athenz/pom.xml

```diff
@@ -109,12 +109,10 @@
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpcore</artifactId>
-            <scope>compile</scope>
```

Comment:
Why is the scope element removed?

Reply:
Because compile is the default, and hence redundant.